tests/endtoend/opart/comparison/forward_time_opart.py

```diff
@@ -0,0 +1,86 @@
+from exojax.spec.opacalc import OpaPremodit
+from exojax.spec.opart import OpartEmisPure
+from exojax.spec.layeropacity import single_layer_optical_depth
+from exojax.utils.grids import wavenumber_grid
+from exojax.spec.api import MdbExomol
+from exojax.utils.astrofunc import gravity_jupiter
+import time
+import tqdm
+import jax.profiler
+import jax.numpy as jnp
+from jax import config
+config.update("jax_enable_x64", True)
+from jax_smi import initialise_tracking
+initialise_tracking()
+
+class OpaLayer:
+    # user defined class, needs to define self.nugrid
+    def __init__(self, Nnus=100000):
+        self.nu_grid, self.wav, self.resolution = wavenumber_grid(
+            1900.0, 2300.0, Nnus, unit="cm-1", xsmode="premodit"
+            #2050.0, 2150.0, Nnus, unit="cm-1", xsmode="premodit"
```

Comment:
```suggestion
#            1900.0, 2300.0, Nnus, unit="cm-1", xsmode="premodit"
            2050.0, 2150.0, Nnus, unit="cm-1", xsmode="premodit"
```
I think it would be better to use a high-resolution grid for the reason of #545